assignments/boolean-birds/code.js

```diff
@@ -11,8 +11,31 @@
  * Diet:
  *
  *   - Flobby Birds eat fish, nuts, and worms.
- *   - Bloggy Birds eat mice, nuts, and worms.
- *   - Flibble Birds eat fish, mice, and worms.
+ *  *   - Flibble Birds eat fish, mice, and worms.
  *   - Globby Birds eat fish, mice, and nuts.
+ *   - Bloggy Birds eat mice, nuts, and worms.
+ * 
+
  *
  */
+const isFlobbyBird=(red, spotted) => red && spotted
+const isBloggyBird=(red, spotted) => red &&! spotted
```

Comment:
```suggestion
const isBloggyBird = (red, spotted) => red && !spotted;
```

The `!` is one of the few operators that doesn't get  a space between it and it's operand.